src/env/mod.rs

```diff
@@ -2,14 +2,35 @@ use crate::api::customization::Customization;
 use crate::api::firmware_protection::FirmwareProtection;
 use crate::api::upgrade_storage::UpgradeStorage;
 use crate::ctap::status_code::Ctap2StatusCode;
-use crate::ctap::Channel;
+use crate::ctap::{Channel, Transport};
 use persistent_store::{Storage, Store};
 use rng256::Rng256;
 
+#[cfg(feature = "std")]
+pub mod host;
 #[cfg(feature = "std")]
 pub mod test;
 pub mod tock;
 
+pub enum SendOrRecvStatus {
+    // FIXME: Replace Option<SendOrRecvStatus>::None with SendOrRecvStatus::TimedOut?
+    Error,
+    Sent,
+    Received(Transport),
```

Comment:
If we have one channel per HID usage page (Main vs. Vendor), we don't need to use `Transport`.